envconfig/tests/test_config_item.py

```diff
@@ -13,16 +13,16 @@
 
 @raises(ValueError)
 def test_config_item_non_string_id():
-    r"""Test config item with non string id"""
+    r"""repronim.envconfig.test_config_item.test_config_item_non_string_id"""
```

Comment:
Remove the "r" in the strings and if the docs are empty, this is what nosetest automatically creates